resources/Examples/compact_osw/measures/PythonPluginPython/measure.py

```diff
@@ -0,0 +1,132 @@
+import os, sys
```

Comment:
```suggestion
import os
import sys
```

Generally speaking you should `pip install black isort` on your system and reformat your python files via

```shell
isort .
black -l 120 .
```